### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by Pawci0**
image

### CODE REVIEW

File: pandemania/public/icons/login-white-24dp.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" viewBox="0 0 24 24" fill="white" width="24px" height="24px"><g><rect fill="none" height="24" width="24"/></g><g><path d="M11,7L9.6,8.4l2.6,2.6H2v2h10.2l-2.6,2.6L11,17l5-5L11,7z M20,19h-8v2h8c1.1,0,2-0.9,2-2V5c0-1.1-0.9-2-2-2h-8v2h8V19z"/></g></svg>
```
Comment (jurczewski) [inline pandemania/public/icons/login-white-24dp.svg]: [Should]
Keep application favicons separate from favicons. Can we just use some font awesome or other icons without downloading them?

File: pandemania/src/components/Navbar/LoginButton/LoginButton.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import IconButton from '@material-ui/core/IconButton';
+import SvgIcon from '@material-ui/core/SvgIcon';
+import * as PropTypes from 'prop-types';
+import React from 'react';
+
+const LoginButton = ({ onClick }) => (
+	<IconButton onClick={onClick} style={{ marginLeft: 'auto' }}>
+		<SvgIcon>
+			<path d="M11,7L9.6,8.4l2.6,2.6H2v2h10.2l-2.6,2.6L11,17l5-5L11,7z M20,19h-8v2h8c1.1,0,2-0.9,2-2V5c0-1.1-0.9-2-2-2h-8v2h8V19z" />
```
Comment (jurczewski) [inline pandemania/src/components/Navbar/LoginButton/LoginButton.jsx:9]: image
[Could]
Is it possible to make them more different?

File: pandemania/src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+					</Typography>
+				</div>
+				{userContext.user == null ? (
+					<LoginButton onClick={() => history.push('/login')} />
+				) : (
+					<LogoutButton onClick={() => history.push('/logout')} />
+				)}
+			</Toolbar>
+		</AppBar>
+	);
```
Comment (jurczewski) [inline pandemania/src/components/Navbar/Navbar.jsx:39]: image
[Could]
Maybe we should make title centered?

File: pandemania/public/icons/login-white-24dp.svg (hunk 1 above)
Comment (Pawci0) [reply pandemania/public/icons/login-white-24dp.svg]: moved to public/assets

File: pandemania/src/components/Navbar/LoginButton/LoginButton.jsx (hunk 2 above)
Comment (Pawci0) [reply pandemania/src/components/Navbar/LoginButton/LoginButton.jsx:9]: added a menu for logged in user

File: pandemania/src/components/Navbar/Navbar.jsx (hunk 3 above)
Comment (Pawci0) [reply pandemania/src/components/Navbar/Navbar.jsx:39]: centered

File: pandemania/src/components/Navbar/LoginButton/LoginButton.jsx (hunk 2 above)
Comment (jurczewski) [reply pandemania/src/components/Navbar/LoginButton/LoginButton.jsx:9]: image
Consider moving it little lower

**Comment by jurczewski** [comment]
image
Wait till my PR Favorite Country Icon is merged, then use country in userContext to display it here.

**Comment by jurczewski** [comment]
image
Navbar is present at loading splashscreen

**Pull request merged**